clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/WgsCramTests.scala

```diff
@@ -479,21 +519,50 @@ trait WgsCramTests { self: BaseIntegrationSpec =>
       _ <- runUpsertCram(key, metadata)
       _ <- runIgnore(ClioCommand.moveWgsCramName, args: _*)
     } yield {
-      Seq(alignmentMetricsDestination, cramSource, craiSource)
-        .foreach(_ shouldNot exist)
+      Seq(
+        cramSource,
+        craiSource,
+        analysisFilesSource,
+        alignmentSummaryMetricsSource,
+        cramValidationReportSource,
+        crosscheckSource,
+        fingerprintSummaryMetricsSource,
+        fingerprintDetailMetricsSource,
+        preAdapterSummaryMetricsSource,
+        preAdapterDetailMetricsSource,
+        preBqsrSelfSmSource
+      ).foreach(_ shouldNot exist)
 
-      Seq(alignmentMetricsSource, cramDestination, craiDestination)
-        .foreach(_ should exist)
+      Seq(
+        cramDestination,
+        craiDestination,
+        analysisFilesDestination,
+        alignmentSummaryMetricsDestination,
+        cramValidationReportDestination,
+        crosscheckDestination,
+        fingerprintSummaryMetricsDestination,
+        fingerprintDetailMetricsDestination,
+        preAdapterSummaryMetricsDestination,
+        preAdapterDetailMetricsDestination,
+        preBqsrSelfSmDestination
+      ).foreach(_ should exist)
 
       // We don't deliver fingerprinting metrics for now because they're based on unpublished research.
-      fingerprintMetricsSource should exist
-      fingerprintMetricsDestination shouldNot exist
+//      fingerprintMetricsSource should exist
```

Comment:
Need to modify the Cram delivery executor to not deliver the fingerprints (which I don't think it does anyway?).
In any case, I think the comment is inappropriate here as we *can* move fingerprints

Reply:
Cram delivery only moves the cram + crai, and generates the md5. It should stay that way (at least in this PR).

Reply:
check